src/java.base/share/classes/java/nio/Direct-X-Buffer.java.template

```diff
@@ -72,6 +74,8 @@ class Direct$Type$Buffer$RW$$BO$
     // Cached unaligned-access capability
     static final boolean UNALIGNED = Bits.unaligned();
 
+    private @Stable boolean temporary; // defaults to false
```

Comment:
Is this used now?

Reply:
No. I removed it in a version I did not check in but forgot here.

Reply:
Changed in [9d84aa9](https://github.com/openjdk/jdk/pull/22339/commits/9d84aa9cfd718265c72e26eff9434f8817d25d80).